tidy3d/components/simulation.py

```diff
@@ -2287,9 +2290,23 @@ def get_neighboring_media(
             new_bounds = (center - dls[0] / 2, center + dls[1] / 2)
             new_geometry = set_bounds(structure.geometry, bounds=new_bounds, axis=axis)
 
+            # check if a regular medium is overlaying a 2D medium
+            media = self.intersecting_media(new_geometry, self.structures[istruct:])
```

Comment:
I don't think this is the desired behavior here. For example, this is not how we treat overlapping 3d materials. A better fix is to just add the line `new_structures.append(structure)` after the line `background_structures.append(structure)`, and then just return `Tuple(new_structures)` at the end of this function.

Reply:
Sorry, I don't think I follow. My issue here is that we generally do not a structure that may even partially overlap with the 2d medium. For example, say I place a graphene sheet, and then an air cylinder on top of it, as I want to make a hole in it. This is not supported, but it is allowed in the current state of things. Are you imagining a structure that *completely* covers the 2D material, and so suggesting to just not volumetrize it?

Now that I wrote this, I think maybe I do see what you may be driving at: we *could* support this, if we would just not re-order the structures list. I think that is actually true. We should have a test for it though.